### PR TITLE
Add Ronin link to Apply section

### LINKED ISSUE #368: Add link to Ronin
jordan [11:06 AM] 
@channel: Guys I’ve been running an experiment over the last couple of weeks linking to Ronin from the MA landing page, and according to Otimizely it hasn’t had a negative affect. Could one of you add it properly now please?

### PR DESCRIPTION
**Pull request by spike01**
Closes #368

### CODE REVIEW

File: source/partials/_apply.html.haml [hunk 1]
```diff
@@ -28,7 +28,9 @@
           = link_to "Payment Options", "/payment.html", class: "button button--horizontal"
           = link_to "Apply", config.apply_form_url, class: "button button--vertical button--large"
           = link_to "Hire Graduates", "/employers.html", class: "button button--horizontal"
-
+        %p
+          Can't join us on a course in London? You might be interested in our online course -
+          = link_to "Makers Academy Remote - codenamed Ronin", "http://ronin.makersacademy.com"
```
Comment (pitchinvasion) [inline source/partials/_apply.html.haml]: I think it's `code named`

Comment (spike01) [inline source/partials/_apply.html.haml]: It's either `codenamed` or `code-named`

File: source/partials/_apply.html.haml [hunk 2]
```diff
@@ -50,3 +52,6 @@
             = link_to "See Our Curriculum", "/curriculum.html", class: "button button--horizontal"
           - else
             = link_to "Learn More", "/about-us.html", class: "button button--horizontal"
+        %p
+          Can't join us on a course in London? You might be interested in our online course -
```
Comment (pitchinvasion) [inline source/partials/_apply.html.haml]: Ditto here, also external links should be target blank

File: source/partials/_apply.html.haml (hunk 1 above)
Comment (pitchinvasion) [inline source/partials/_apply.html.haml]: neither of those are in the dictionary, but meh

Comment (spike01) [inline source/partials/_apply.html.haml]: https://en.wiktionary.org/wiki/codename#English

Comment (pitchinvasion) [inline source/partials/_apply.html.haml]: code-named looks more right though

**Comment by pitchinvasion** [comment]
:shipit: your call on my comments

File: source/partials/_apply.html.haml (hunk 1 above)
Comment (spike01) [inline source/partials/_apply.html.haml]: image

Comment (pitchinvasion) [inline source/partials/_apply.html.haml]: i speak english more right than you

Comment (pitchinvasion) [inline source/partials/_apply.html.haml]: http://dictionary.cambridge.org/spellcheck/english/?q=codename

**Pull request merged**